### ISSUE TITLE
Compiling fails with GCC 6.2.0

### ISSUE THREAD

**Issue by lckarssen** [GenABEL-Project/ProbABEL#32]
I tried to compile ProbABEL on Ubuntu 16.10 with GCC 6.2.0 and it failed with the following warning:

```
In file included from main.cpp:73:0:
reg1.h: At global scope:
reg1.h:188:31: error: ‘constexpr’ needed for in-class initialization of static data member ‘const double logistic_reg::EPS’ of non-integral type [-fpermissive]
     static const double EPS = 1e-8;
                               ^~~~
In file included from main.cpp:75:0:
coxph_data.h:223:31: error: ‘constexpr’ needed for in-class initialization of static data member ‘const double coxph_reg::EPS’ of non-integral type [-fpermissive]
     static const double EPS = 1e-9;
                               ^~~~
coxph_data.h:229:35: error: ‘constexpr’ needed for in-class initialization of static data member ‘const double coxph_reg::CHOLTOL’ of non-integral type [-fpermissive]
     static const double CHOLTOL = 1.5e-12;
                                   ^~~~~~~
```

**Comment by lckarssen** [comment]
Fixed with commit 2267200.

**Comment by lckarssen** [comment]
Hmm, now the Jenkins build (on Ubuntu 16.04, GCC 5.4.0) fails...

**Comment by lckarssen** [comment]
Fixed with PR #35.
